### DISCUSSION TITLE
Swedish Characters on LCD

### DISCUSSION THREAD

**Discussion by WattageGuy** [micropython/micropython#9343] [General]
Hello, I can't display Swedish characters like äåö on my LCD. I've tried a bunch of LCD libraries and none of them display the right characters. I'm using i2c bus and I don't exactly know what screen or interface other than it is 16x2. I've tried decode string into UTF-8 but it doesn't help. Is my LCD the problem? Please help.

**Comment by rolandvs** [comment]
If your display is an HD44780 like displays you can show custom characters. For an example on how to do this look at this site: https://www.engineersgarage.com/making-custom-characters-on-lcd-using-arduino/.

**Reply by WattageGuy** [reply]
Okay thanks, I'll try. In other terms åäö is not natively supported?

**Reply by scruss** [reply]
Maybe; depends on the character ROM on your board. [page 18 of the datasheet](https://www.sparkfun.com/datasheets/LCD/HD44780.pdf) shows the A02 layout. These displays don't support Unicode. They only understand 8-bit byte character codes in their own encoding
